test/tooltip-windowResize.html

```diff
@@ -0,0 +1,383 @@
+                        }
+                    },
+                    splitLine: {
+                        show: false
+                    }
+                },
+                yAxis: {
+                    type: 'value',
+                    axisTick: {
+                        inside: true
+                    },
+                    splitLine: {
+                        show: false
+                    },
+                    axisLabel: {
+                        inside: true,
+                        formatter: '{value}\n'
+                    },
+                    z: 10
+                },
+                grid: {
+                    top: 110,
+                    left: 15,
+                    right: 15,
+                    height: 160
+                },
+                dataZoom: [{
+                    type: 'inside',
+                    throttle: 50
+                }],
+                series: [{
+                        name: '模拟数据',
+                        type: 'line',
+                        smooth: true,
+                        symbol: 'circle',
+                        symbolSize: 5,
+                        sampling: 'average',
+                        itemStyle: {
+                            color: '#8ec6ad'
+                        },
+                        stack: 'a',
+                        areaStyle: {
+                            color: new echarts.graphic.LinearGradient(0, 0, 0, 1, [{
+                                offset: 0,
+                                color: '#8ec6ad'
+                            }, {
+                                offset: 1,
+                                color: '#ffe'
+                            }])
+                        },
+                        data: data
+                    },
+                    {
+                        name: '模拟数据',
+                        type: 'line',
+                        smooth: true,
+                        stack: 'a',
+                        symbol: 'circle',
+                        symbolSize: 5,
+                        sampling: 'average',
+                        itemStyle: {
+                            color: '#d68262'
+                        },
+                        areaStyle: {
+                            color: new echarts.graphic.LinearGradient(0, 0, 0, 1, [{
+                                offset: 0,
+                                color: '#d68262'
+                            }, {
+                                offset: 1,
+                                color: '#ffe'
+                            }])
+                        },
+                        data: data2
+                    }
+
+                ]
+            };
+
+            var chart = testHelper.create(echarts, 'main1', {
+                option: option
+            });
+            chart.setOption(option, true);
+            window.onresize = function () {
+                chart.resize();
+            }
```

Comment:
Consider using `window.addEventListener`, `window` is shard with these two test cases, `window.onresize` in the bottom test case will override the top one.

Reply:
> Consider using `window.addEventListener`, `window` is shard with these two test cases, `window.onresize` in the bottom test case will override the top one.

ok, i see, thank you!